airflow/providers/amazon/aws/example_dags/example_emr_serverless.py

```diff
@@ -0,0 +1,97 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
new example dags should be according to AIP-47
Example: https://github.com/apache/airflow/pull/25205

Reply:
That's a good point but the plan was to do it in two iterations since this PR is already big enough. Once this PR is merged, @syedahsn will work on converting this sample dag to system test. If you strongly disagree, I guess the two can be done in this PR but it will just make it bigger and harder to review